### PR TITLE
enh(admin): page titles, fields name, translations and help

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

Review Gorgone administration page titles, fields name, translations and help

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -4303,10 +4306,8 @@ msgstr "Modifier les options de Centcore"
 
 #: centreon-web/www/include/Administration/parameters/api/api.php:61
 #: centreon-web/www/include/Administration/parameters/centcore/centcore.php:59
-msgid "Enable Broker Statistics Collection"
-msgstr ""
-"Activer la récupération des fichiers de statistiques des composants Centreon "
-"Broker"
+msgid "Enable Broker statistics collection"
+msgstr "Activer la récupération des fichiers de statistiques des composants Broker"
```
Comment (sc979) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]: Maybe too long

Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Activer la récupération des fichiers de statistiques Broker"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -4317,6 +4318,15 @@ msgstr "Temps maximum d'exécution des commandes Gorgone"
...
 msgstr "Caractères illégaux pour les commandes Gorgone"
 
+msgid "IP address or hostname"
+msgstr "Adresse IP ou nom d'hôte"
+
+msgid "Use SSL/TLS"
+msgstr "Utiliser SSL/TLS"
+
+msgid "Allow self signed certificate"
+msgstr "Autorise les certificats auto-signés"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Autoriser les certificats auto-signés"
```

File: www/include/Administration/parameters/gorgone/help.php [hunk 3]
```diff
@@ -4,26 +4,48 @@
...
 $help['tip_gorgone_cmd_timeout'] = dgettext(
     'help',
     "Timeout value in seconds. Used to make actions calls timeout."
 );
 $help['tip_enable_broker_stats'] = dgettext(
     'help',
-    'Enable Centreon Broker statistics collection into the central server in order to reduce'
-    . ' the network load generated by Gorgone. Be carefull: broker statistics will not be'
+    'Enable Centreon Broker statistics collection into the central server.'
+    . ' Be carefull: if disabled, Broker statistics will not be'
```
Comment (kduret) [inline www/include/Administration/parameters/gorgone/help.php]:
```suggestion
    . ' Be careful: if disabled, Broker statistics will not be'
```

**Pull request merged**